exec/java-exec/src/test/java/org/apache/drill/exec/expr/fn/registry/FunctionRegistryHolderTest.java

```diff
@@ -155,6 +162,47 @@ public void testGetAllJarNames() {
     compareTwoLists(expectedResult, registryHolder.getAllJarNames());
   }
 
+  @Test
+  public void testGetAllJarsWithFunctionHolders() {
+    Map<String, List<FunctionHolder>> fnHoldersInRegistry = registryHolder.getAllJarsWithFunctionHolders();
+    //Iterate and confirm lists are same
+    for (String jarName : Lists.newArrayList(newJars.keySet())) {
```

Comment:
No need to wrap into ArrayList here and a couple more times below in the code.

Reply:
Done